python/sglang/srt/server_args.py

```diff
@@ -2973,6 +2973,8 @@ def model_specific_adjustments(self):
                 self.attention_backend = "nsa"
                 logger.warning("Set nsa attention backend for DeepSeek NSA.")
 
+            if is_npu():
+                return
```

Comment:
maybe do not "return", o/w affect potential future code. maybe if not is_npu(): ...

Reply:
Right, thanks, we have fixed it.